### DISCUSSION TITLE
2011/01/17/%E5%A4%A7%E7%88%86%E7%82%B8%E7%9A%84%E9%80%9F%E5%BA%A6

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#311] [Announcements]
# 2011/01/17/%E5%A4%A7%E7%88%86%E7%82%B8%E7%9A%84%E9%80%9F%E5%BA%A6

刚刚看一条新闻，里边提到说现在科学家观测到的最远的星系距离我们有130亿光年。但我有个地方始终没想明白。

https://blog.qizhen.xyz/2011/01/17/%E5%A4%A7%E7%88%86%E7%82%B8%E7%9A%84%E9%80%9F%E5%BA%A6

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Yang
2011年01月27日 05:33 编辑
宇宙是膨胀的，130亿光年外的光，并不需要130亿年才能传到地球。假设，这星星的光是在5亿光年外发出来的， 它传过来可能需要5亿年，但是宇宙膨胀了，5亿年没传过来，可能花了20亿年，但是等它传过来的时候，原来的星星和地球之间的距离已经膨胀到了130亿光年。简单来说，就是这个解释。复杂的数学论证我就不会了。

回复
